jfrog_platform_cloud/README.md

```diff
@@ -20,6 +20,88 @@ These logs will allow customers to readily know who accessed what repositories a
 
 JFrog's SaaS Log Streaming is currently in beta. While in beta, the cloud log streaming feature will only be available inside the MyJFrog portal for select JFrog Enterprise and customers. JFrog plans to GA this feature later in Q2 2024 at which point it will be available to all JFrog Enterprise and customers.
 
+## Widgets description
```

Comment:
Can you provide more insight on the changes? Is this referring to dashboard widgets?